agent/containers/images/visualizers/prom-graf-visualizer/prom_ds.py

```diff
@@ -0,0 +1,65 @@
+import json
+import requests
+import subprocess
+import time
+
+graf_base = "http://localhost:3000/"
```

Comment:
Again, an inline description of function and purpose, combined with a more specific name, would help this file a lot. It does at least seem to be distinct from its counterpart.